ParseCSV/ParseCSV/Validator.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace ParseCSV
+{
+    static class Validator
+    {
+        static StringBuilder path = new StringBuilder();
+
+        public static bool IsPathInputFileCorret(string inputPath)
+        {
+            path.Clear();
+            var pathArray = inputPath.Split('\\');
+            for (int i = 0; i < pathArray.Length - 1; i++)
+            {
+                path.Append(pathArray[i] + "\\");
+            }
+
+            return (Directory.Exists(path.ToString()) && new FileInfo(inputPath).Exists) ? true : false;
+        }
+
+        public static (bool isNameCorrect, bool isPathCorrect) IsPathOutputFileCorret(string inputPath)
```

Comment:
Here I suggest you to try to use more robust approach, as an example: 
```
public class ValidationResult
{
public bool Result {get; set; } // as alternative this can be an enum with values Success and Failure
public string Message {get; set;}
// other properties if needed
}
```

Reply:
Do you want get rid of tuple?  I guessed about it, but I still made it with tuple. I think, here it looks good.

Reply:
In terms of design class will look better.
Please take a look at `ValidationResult `class. It's instance can carry a simple boolean Result value - it aggregates validation result, you don't need to check if both `bool isNameCorrect, bool isPathCorrect` are `true`.
`Message ` will say what's not correct, you can display/write in log it when needed.
In real life such classes can carry exception stack trace, which can be useful in some cases.

Let's use tuples when they can simplify code that is written using classes (and do not hurt anything else).